pkg/operator2/operator.go

```diff
@@ -62,7 +62,8 @@ const (
 	kasServiceAndEndpointName = "kubernetes"
 	kasServiceFullName        = kasServiceAndEndpointName + "." + corev1.NamespaceDefault + ".svc"
 
-	rootCAFile = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	systemTrustStoreDirPath = "/etc/pki/ca-trust/extracted/pem"
```

Comment:
No.  `rootCAFile` is for in-cluster communication.

And even if you are trying to say "use this single bundle for outgoing communication" you will quickly run into the issue that sometimes you need a real CA from the default root store and sometimes you need the `tls-ca-bundle.pem` thing.

Reply:
@enj see my [comment](https://github.com/openshift/cluster-authentication-operator/pull/172/files#r314894913), `tls-ca-bundle.pem` is a merged user-provided/system trust bundle.

Reply:
I'm wondering whether @enj's comment is maybe more about trust domain separation rather than about the bundle being a universal hammer for trust.

I'm not quite familiar with how CA bundles are loaded for client connections for in-cluster API communication, but I guess if this was an issue, it would be an issue for all the components that mount the bundle such as we do here.

Reply:
Nvm, this is indeed wrong, `rootCAFile` is only used for in-cluster communication and should not be changed.